pkg/backuphelpers/backupvars_test.go

```diff
@@ -0,0 +1,113 @@
+package backuphelpers
+
+import (
+	"testing"
+
+	backupv1alpha1 "github.com/openshift/api/config/v1alpha1"
+	prune "github.com/openshift/cluster-etcd-operator/pkg/cmd/prune-backups"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	schedule = "0 */2 * * *"
+	timezone = "GMT"
+)
+
+func TestBackupConfig_ToArgs(t *testing.T) {
+	testCases := []struct {
+		name     string
+		cr       *backupv1alpha1.EtcdBackupSpec
+		expected string
+	}{
+		{
+			"enabled",
+			createEtcdBackupSpec(timezone, schedule),
+			"    args:\n    - --enabled=true\n    - --timezone=GMT\n    - --schedule=0 */2 * * *",
+		},
+		{
+			"disabled",
```

Comment:
disabled means "nil" now, maybe adjust the test to match this?

Reply:
👍🏽

Reply:
done 👍🏽